guides/release/pages.yml

```
@@ -1,325 +1,325 @@
       url: "index"
-    - title: "Quick Start"
+    - title: "Démarrage rapide"
       url: "quick-start"
-    - title: "Working with HTML, CSS, and JavaScript"
+    - title: "Travailler avec HTML, CSS, et JavaScript"
       url: "working-with-html-css-and-javascript"
-    - title: "Anatomy of an Ember App"
+    - title: "Anatomie d'une App Ember"
       url: "anatomy-of-an-ember-app"
-- title: "Tutorial"
+- title: "Tutoriel"
   url: "tutorial"
   pages:
-    - title: "Part 1"
+    - title: "Partie 1"
       url: "part-1"
       pages:
         - title: "Introduction"
           url: "index"
         - title: "Orientation"
           url: "orientation"
-        - title: "Building Pages"
+        - title: "Construire des Pages"
           url: "building-pages"
-        - title: "Automated Testing"
+        - title: "Tests Automatisés"
           url: "automated-testing"
-        - title: "Component Basics"
+        - title: "Composants Basiques"
           url: "component-basics"
-        - title: "More About Components"
+        - title: "Plus sur les Composants"
           url: "more-about-components"
-        - title: "Interactive Components"
+        - title: "Composants Interactifs"
           url: "interactive-components"
-        - title: "Reusable Components"
+        - title: "Composants Réutilisables"
           url: "reusable-components"
-        - title: "Working With Data"
+        - title: "Travailler avec des Données"
           url: "working-with-data"
-        - title: "Recap"
+        - title: "Récap"
           url: "recap"
-    - title: "Part 2"
+    - title: "Partie 2"
       url: "part-2"
       pages:
         - title: "Introduction"
           url: "index"
-        - title: "Route Params"
+        - title: "Paramètres des Routes"
           url: "route-params"
-        - title: "Service Injection"
+        - title: "Injection de Service"
           url: "service-injection"
         - title: "Ember Data"
           url: "ember-data"
-        - title: "Provider Components"
+        - title: "Composants Provider"
           url: "provider-components"
-        - title: "Recap"
+        - title: "Récap"
           url: "recap"
 
 - title: "Core Concepts"
   url: "toc-heading_core-concepts"
   is_heading: true
-- title: "Components"
+- title: "Composants"
   url: components
   pages:
-    - title: "Templates are HTML"
+    - title: "Les Templates sont du HTML"
       url: "index"
-    - title: "Introducing Components"
+    - title: "Introduction aux Composants"
       url: "introducing-components"
-    - title: "Component Arguments and HTML Attributes"
+    - title: "Arguments des Composants et Attributs HTML"
       url: "component-arguments-and-html-attributes"
-    - title: "Conditional Content"
+    - title: "Contenu Conditionnel"
       url: "conditional-content"
-    - title: "Block Content"
+    - title: "Contenu des Blocs"
       url: "block-content"
-    - title: "Helper Functions"
+    - title: "Fonctions Helper"
```

I suppose we want to keep the root word `Helper` right?

Like before, what about:

```suggestion
    - title: 'Fonctions dit "Helpers"'
```

---

```suggestion
    - title: "Fonctions dites "Helpers""
```